internal/configs/sharding/mainnet.go

```diff
@@ -57,10 +64,20 @@ func (ms mainnetSchedule) IsLastBlock(blockNum uint64) bool {
 	}
 }
 
-var mainnetReshardingEpoch = []*big.Int{big.NewInt(0), big.NewInt(mainnetV1Epoch), big.NewInt(mainnetV2Epoch)}
+func (ms mainnetSchedule) VdfDifficulty() int {
+	return mainnetVdfDifficulty
+}
+
+// ConsensusRatio ratio of new nodes vs consensus total nodes
+func (ms mainnetSchedule) ConsensusRatio() float64 {
+	return mainnetConsensusRatio
+}
+
+var mainnetReshardingEpoch = []*big.Int{big.NewInt(0), big.NewInt(mainnetV1Epoch), big.NewInt(mainnetV2Epoch), big.NewInt(mainnetV3Epoch)}
 var mainnetV0 = MustNewInstance(4, 150, 112, genesis.HarmonyAccounts, genesis.FoundationalNodeAccounts, mainnetReshardingEpoch)
 var mainnetV1 = MustNewInstance(4, 152, 112, genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV1, mainnetReshardingEpoch)
 var mainnetV2 = MustNewInstance(4, 200, 148, genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV2, mainnetReshardingEpoch)
+var mainnetV3 = MustNewInstance(4, 248, 138, genesis.HarmonyAccounts, genesis.FoundationalNodeAccountsV3, mainnetReshardingEpoch)
```

Comment:
Really.  248 per shard? It is wrong.  you added 40 more keys.  the number should be 210.  Also, why do you change the number of harmony nodes to 138?  it should be 148, no change.